win32ss/user/ntuser/winpos.c

```diff
@@ -3727,17 +3722,20 @@ NtUserSetWindowPlacement(HWND hWnd,
    }
    _SEH2_END
 
-   if(Safepl.length != sizeof(WINDOWPLACEMENT))
-   {
-      goto Exit; // Return FALSE
-   }
-
-   Flags = PLACE_MAX | PLACE_RECT;
-   if (Safepl.flags & WPF_SETMINPOSITION) Flags |= PLACE_MIN;
-   UserRefObjectCo(Wnd, &Ref);
-   IntSetWindowPlacement(Wnd, &Safepl, Flags);
-   UserDerefObjectCo(Wnd);
-   Ret = TRUE;
+    UserRefObjectCo(Wnd, &Ref);
```

Comment:
@katahiromz Could you move this UserRef/Deref within the if-block, where the Wnd is actually used?

Reply:
Moved.